app/models/case_contact.rb

```diff
@@ -240,7 +240,7 @@ def contact_groups_with_types
   end
 
   def requested_followup
-    followups.requested.first
+    followups.find(&:requested?)
```

Comment:
We are including `followups` in the query already, however, the call to `.requested` forces another query `Followups.where(case_id: id, status: 0)` because it needs to check the status. Since all the data is already loaded it makes more sense to check that status in ruby rather than making another request.